ports/libhv/vcpkg.json

```
@@ -0,0 +1,6 @@
+{
+  "name": "libhv",
+  "version-string": "v1.0.0",
```

```suggestion
  "version": "1.0.0",
```
Please see the doc https://github.com/microsoft/vcpkg/blob/master/docs/specifications/versioning.md#22-package-versions to learn more about package versions.